### PR TITLE
P3 crafted shoulder item effects

### PR DESCRIPTION
**Pull request by tyler-cb**
https://www.wowhead.com/classic/news/new-phase-3-profession-items-coming-to-classic-season-of-discovery-338284

### CODE REVIEW

**Comment by kayla-glick** [comment]
https://github.com/wowsims/sod/issues/516

**Review by kayla-glick — APPROVED** [review]
We could break these `Echoes of X` auras out into functions in this file to save on repetition since a lot of them are shared by multiple items but I wouldn't consider that a blocker. Just means a little less code and less chance for typos 😄

File: sim/common/sod/crafted/phase_3.go [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+		})
+
+		handler := func(sim *core.Simulation, _ *core.Spell, _ *core.SpellResult) {
+			procAura.Activate(sim)
+		}
+
+		core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
+			ActionID:   core.ActionID{SpellID: 446579},
+			Name:       "Echoes of Dread",
+			Callback:   core.CallbackOnCastComplete,
```
Comment (rosenrusinov) [inline sim/common/sod/crafted/phase_3.go:150]: core.CallbackOnSpellHitDealt

Comment (rosenrusinov) [reply sim/common/sod/crafted/phase_3.go:150]: Actually just checked wowhead and im wrong. It  should be cast complete

**Review by rosenrusinov — APPROVED** [review]

**Pull request merged**